pyiron_atomistics/atomistics/structure/analyse.py

```diff
@@ -47,6 +76,272 @@ def get_average_of_unique_labels(labels, values):
+                initiate the symmetrization of interstitial candidate sites.
+
+        Returns:
+            ( (n, 3) numpy.ndarray) Interstitial candidate positions.
+
+        This function internally does the following steps:
+
+            1. Initialize grid points (or Voronoi vertices) which are considered as
+                interstitial site candidates.
+            2. Eliminate points within a distance from the nearest neighboring atoms as
+                given by `min_distance`
+            3. Initialize neighbor environment using `get_neighbors`
+            4. Shift interstitial candidates to the nearest symmetric points with respect to the
+                neighboring atom sites/vertices.
+            5. Kick out points with large neighbor distance variances; this eliminates "irregular"
+                shaped interstitials
+            6. Cluster interstitial candidates to avoid point overlapping.
+
+        In complex structures (i.e. grain boundary, dislocation etc.), the default parameters
+        should be chosen properly. In order to see other quantities, which potentially
+        characterize interstitial sites, see the following functions from
+        `structure.analyse.interstitials`:
+
+            - `get_variance()`
+            - `get_distance()`
+            - `get_steinhardt_parameter()`
+            - `get_volume()`
+            - `get_area()`
+        """
+        self.num_neighbors = num_neighbors
+        for _ in range(n_iterations):
+            self._set_interstitials_to_high_symmetry_points()
+        self._kick_out_points(variance_buffer=variance_buffer)
+        self._cluster_points(eps=eps)
+        return self.positions
+
+    def get_variance(self):
+        """
+        Get variance of neighboring distances. Since interstitial sites are mostly in symmetric
+        sites, the variance values tend to be small. In the case of fcc, both tetrahedral and
+        octahedral sites as well as tetrahedral sites in bcc should have the value of 0.
+
+        Returns:
+            (numpy.array (n,)) Variance values
+        """
+        return np.std(self.neigh.distances, axis=-1)
+
+    def get_distance(self, function_to_apply=np.min):
+        """
+        Get per-position return values of a given function for the neighbors.
+
+        Args:
+            function_to_apply (function): Function to apply to the distance array. Default is
+                numpy.minimum
+
+        Returns:
+            (numpy.array (n,)) Function values on the distance array
+        """
+        return function_to_apply(self.neigh.distances, axis=-1)
+
+    def get_steinhardt_parameter(self, l):
+        """
+        Args:
+            l (int/numpy.array): Order of Steinhardt parameter
+
+        Returns:
+            (numpy.array (n,)) Steinhardt parameter values
+        """
+        return self.neigh.get_steinhardt_parameter(l=l)
+
+    def get_volume(self):
```

Comment:
I thought about it as well, but then I decided to use `get_` anyway, because all properties related to characterising interstitial sites are obtained via `get_`. And in the current implementation it's difficult to live without them. So I guess I'd rather keep the current one.

In light of `Interstitials(Atoms)`, I'm gonna change this maybe to plural to avoid conflicts...